main.scss

```
@@ -53,6 +53,14 @@
 .ncf {
 	@include oTypographySans($scale: 0);
 
+	&__divider-horizontal {
+		margin: 0 0 16px;
+		height: 0px;
```

```suggestion
		height: 0;
```
unit is optional for 0